### PR TITLE
feature: adding about us section

### PR DESCRIPTION
**Pull request by Jaciubek**
feature about has been added

### CODE REVIEW

**Review by mateuszkierylo — APPROVED** [review]

File: src/css/about.css [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+.about-marker::after {
+    content: "";
+    position: absolute;
+    display: block;
+    top: 29px;
+    right: -2px;
+    width: 41px;
+    height: 12px;
+    border-radius: 8px;
+    background-color: #BEE8CC;
```
Comment (PiotrJasinski1995) [inline src/css/about.css:32]: why not used variables

File: src/css/about.css [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    background-color: var(--green-color);
+}
+
+/* Styles for Tablet devices */
+@media only screen and (min-width: 768px) {
+    .about-card-container {
+        padding: 0 23px;
+    }
+
+    .about-heder {
```
Comment (PiotrJasinski1995) [inline src/css/about.css:112]: "header", not "heder"

File: src/css/about.css [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        font-size: 40px;
+        line-height: 1.2;
+        letter-spacing: -0.02em;
+    }
+    
+    .about-marker::after {
```
Comment (PiotrJasinski1995) [inline src/css/about.css:118]: look how the element is positioned, different views (mobile, tablet, desktop), different position

File: src/partials/about.html [hunk 4]
```diff
@@ -1,3 +1,62 @@
...
+          <div class="about-list-item-container">
+            <h3 class="about-list-header">04</h3>
+            <p class="about-list-paragraph">
+              Comprehensive language support services
+            </p>
+          </div>
+        </li>
+      </ul>
+      <div class="about-picture">
+        <picture>
```
Comment (PiotrJasinski1995) [inline src/partials/about.html:46]: use just  with source set, not picture (you can look in "proposal to leave app" section

File: src/partials/about.html [hunk 5]
```diff
@@ -1,3 +1,62 @@
...
+          <source
+            srcset="
+              ../img/memoji.png 1x,
+              ../img/memoji@2x.png 2x
+            "
+          />
+          <img
+            class="about-picture-img"
+            src="../img/memoji.png"
```
Comment (PiotrJasinski1995) [inline src/partials/about.html:55]: wrong path, it should be "./", not "../", check where the index.html file is located relative to the images, not your file

**Review by PiotrJasinski1995 — COMMENTED** [review]
this error should be corrected

**Pull request merged**